#--------------------------------------
#
# Herkulex package custom jog multiple message
#
# used to send jog commands to a set of servos.
#
#--------------------------------------

Jog[] joint_jogs